paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -61,12 +76,34 @@ def add_subparser(subparsers):
         required=True,
     )
     list_parser.add_argument(
+        '--wait-for-deployment',
+        help='Set to poll paasta and wait for the deployment to finish, '
+             'the default strategy is to mark for deployment and exit straightaway',
+        dest='block',
+        action='store_true',
+        default=False
+    )
+    list_parser.add_argument(
+        '-t', '--timeout',
+        dest="timeout",
+        type=int,
+        default=DEFAULT_DEPLOYMENT_TIMEOUT,
```

Comment:
You only really use this constant here, I think it would be ok to not have the indirection if we don't need it